packages/sheets-ui/src/services/clipboard/html-to-usm/converter.ts

```diff
@@ -230,18 +224,63 @@ export class HtmlToUSMService {
                 rowProperties.push(...tableRowProp);
             });
         }
+        this.dispose();
         return {
             rowProperties,
             colProperties,
             cellMatrix: valueMatrix,
         };
     }
 
+    private _getStyleBySelectorText(selectorText: string, cssText: string) {
+        const css = this._styleMap.get(selectorText)?.getPropertyValue(cssText);
+        if (!css) {
+            return '';
+        }
+        return css;
+    }
+
+    // eslint-disable-next-line complexity
+    private _getStyle(node: HTMLElement, styleStr: string) {
+        const recordStyle: Record<string, string> = turnToStyleObject(styleStr);
+        const style = node.style;
+        // style represents inline styles with the highest priority, followed by selectorText which corresponds to stylesheet rules, and recordStyle pertains to inherited styles with the lowest priority.
+        let newStyleStr = '';
+        for (let l = 0; l < sheetStyleRules.length; l++) {
+            const key = sheetStyleRules[l];
+
+            // retrieve multiple sources for a node and compile them into a cohesive new style string.
+            if (key === 'background') {
+                const bgColor = style.getPropertyValue('background-color') || this._getStyleBySelectorText(`.${node.className}`, 'background-color') ||
+                    this._getStyleBySelectorText(`.${node.className}`, key) ||
+                    this._getStyleBySelectorText(node.nodeName.toLowerCase(), key) || this._getStyleBySelectorText(node.nodeName, 'background-color') || recordStyle['background-color'] || '';
+                bgColor && (newStyleStr += `background:${bgColor};`);
+                continue;
+            }
+            if (key === 'text-decoration') {
+                const textDecoration = style.getPropertyValue('text-decoration-line') || style.getPropertyValue('text-decoration') || this._getStyleBySelectorText(`.${node.className}`, 'text-decoration-line') ||
+                    this._getStyleBySelectorText(`.${node.className}`, key) ||
+                    this._getStyleBySelectorText(node.nodeName.toLowerCase(), key) || this._getStyleBySelectorText(node.nodeName, 'text-decoration-line') || recordStyle['text-decoration-line'] || '';
+                textDecoration && (newStyleStr += `text-decoration:${textDecoration};`);
+                continue;
+            }
+
+            const value =
+                style.getPropertyValue(key) ||
+                this._getStyleBySelectorText(`.${node.className}`, key) ||
+                this._getStyleBySelectorText(node.nodeName.toLowerCase(), key) ||
+                recordStyle[key] ||
```

Comment:
here need to consider classList and id，

Reply:
I used Excel, Tencent Docs, and Google Sheets for pasting operations, and I haven't found any cell nodes using multiple class names or ID selectors. However, to truly replace the use of the getComputedStyle API, I will consider this case.